conf/global_zookeeper.conf

```diff
@@ -19,21 +19,21 @@
 
 # The number of milliseconds of each tick
 tickTime=2000
-# The number of ticks that the initial 
+# The number of ticks that the initial
 # synchronization phase can take
 initLimit=10
-# The number of ticks that can pass between 
+# The number of ticks that can pass between
 # sending a request and getting an acknowledgement
 syncLimit=5
 # the directory where the snapshot is stored.
-dataDir=data/global-zookeeper
+dataDir=data/configuration-store
```

Comment:
I think changing this might break BC. it is probably okay to leave the value as "data/global-zookeeper" for now.